src/components/bottom-tab-nav/bottom-tab-nav.tsx

```diff
@@ -54,7 +54,7 @@ export class BottomTabNav extends PureComponent<
     if (!this.state.fontLoaded) {
       return (
         <AppLoading
-          startAsync={Font.loadAsync(customFonts)}
+          startAsync={async () => { await Font.loadAsync(customFonts) }}
```

Comment:
is the `async` and `await` in here necessary? Wouldn't `Font.loadAsync` already return a Promise?